docs/handbook/getting-started.md

```diff
@@ -1230,24 +1230,8 @@ Note the following points about IPS packages:
 Actions are defined when an IPS package is created.
 Actions are used for defining the files and directories of the package, setting package attributes, declaring dependencies on other packages, creating users and groups, and installing device drivers.
 Some actions may optionally have tags that provide meta information about the action such as locale information and debug configuration.
-
-
-### IPS package archives (.p5p)
-
-When the IPS system was originally conceived there was no standard on-disk format for an IPS package.
-Hence, unlike a .rpm file, an SVR4 package, or a .nbm file, it was not possible to transfer an IPS package from system to system.
-The remote IPS repository where the IPS package resided was the only source for the package.
-This is because in its native state, the IPS package is not something you can normally download as a single archive file.
-
-Recognizing this limitation of IPS, the `.p5p` IPS archive format was developed.
-For IPS archives, files are stored in the pax archive format, along with additional metadata, such as IPS manifest files, and a specific layout of the contents.
-The `pkgrecv` command is used to create IPS archives.
-
-For example, to create a `.p5p` IPS package archive containing the package `editor/gnu-emacs` and all of its dependencies from the repository located at `http://example.com:10000`, use the following command:
-
-```bash
-pkgrecv -s http://example.com:10000 -d /my/emacs.p5p -a -r editor/gnu-emacs
-```
+* IPS packages can only be installed from an IPS repository.
+    * IPS package repositories can be local to the system or on a remote networked system.
```

Comment:
Hi Alp,

It was only moved towards the end of the IPS section.

Michael

On Aug 23, 2016 10:05 AM, "Alexander Pyhalov" <notifications@github.com>
wrote:

In docs/handbook/getting-started.md
<https://github.com/OpenIndiana/oi-docs/pull/42#discussion_r75872194>:

> -When the IPS system was originally conceived there was no standard on-disk format for an IPS package.
> -Hence, unlike a .rpm file, an SVR4 package, or a .nbm file, it was not possible to transfer an IPS package from system to system.
> -The remote IPS repository where the IPS package resided was the only source for the package.
> -This is because in its native state, the IPS package is not something you can normally download as a single archive file.
> -
> -Recognizing this limitation of IPS, the `.p5p` IPS archive format was developed.
> -For IPS archives, files are stored in the pax archive format, along with additional metadata, such as IPS manifest files, and a specific layout of the contents.
> -The `pkgrecv` command is used to create IPS archives.
> -
> -For example, to create a `.p5p` IPS package archive containing the package `editor/gnu-emacs` and all of its dependencies from the repository located at `http://example.com:10000`, use the following command:
> -
> -```bash
> -pkgrecv -s http://example.com:10000 -d /my/emacs.p5p -a -r editor/gnu-emacs
> -```
> +* IPS packages can only be installed from an IPS repository.
> +    * IPS package repositories can be local to the system or on a remote networked system.

why did you remove mentions of p5p files?

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/OpenIndiana/oi-docs/pull/42/files/16967556883a9783be366a86f0444a9a14b2d29a#r75872194>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AP9jzHMoeh16Ty145QDsSZUiSpSdwsrQks5qiv4IgaJpZM4Jm3Oi>
.